SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -240,7 +240,7 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 	RuntimeStartTime = FDateTime::Now();
 	bRedeployRequired = false;
 
-	if (bLocalDeploymentRunning)
+	if (bLocalDeploymentRunning || bStartingDeployment)
```

Comment:
Do we also need to worry about a deployment being gracefully stopped here?

Reply:
So the only important callsite to this currently already checks the deployment is stopping. The other one is tests where graceful shutdown is force disabled. Should be good.